Funky.js

```diff
@@ -0,0 +1,50 @@
+const funkyRotate = () => {
+    const flexBox = document.getElementById("FunkyArea");
+    const style = window.getComputedStyle(flexBox)
+    const currentDirection = style.getPropertyValue("flex-direction");
+    
+    if (currentDirection === "row")
+        flexBox.style.flexDirection = "column"
+    else if (currentDirection === "column")
+        flexBox.style.flexDirection = "row"
+}
+const funkyRotate2 = () => {

```

Comment:
lmao yeah you're copy pasting code. Big bad. You should have a single method that does this, where you pass in the element you want. This sort of code would always be denied in PR reviews